build/javascript.mk

```diff
@@ -32,10 +30,7 @@ javascript-rn: apply-patches lua53 milagro embed-lua
 	@cp -v src/zenroom.js 	  build/rnjs/
 
 javascript-demo: cflags  += -DARCH_WASM -D'ARCH=\"WASM\"' -D MAX_STRING=128000
-javascript-demo: ldflags += -s WASM=1 \
-	-s ASSERTIONS=1 \
-	-s NO_EXIT_RUNTIME=1 \
-	--shell-file ${extras}/shell_minimal.html
+javascript-demo: ldflags += -s WASM=1 --shell-file ${extras}/shell_minimal.html
```

Comment:
ask you to leave the ASSERTION=1 for debug purposes. Specially for the demo, that print stuff on the console.err if something goes wrong ;)

Reply:
Good point, I missed the fact that some of the defaults read in https://github.com/emscripten-core/emscripten/blob/master/src/settings.js do change when optimization level is above 0, and `ASSERTIONS` is one of that. I'm going to put it back.